static/src/javascripts/projects/common/modules/identity/identity-features.js

```diff
@@ -0,0 +1,23 @@
+// @flow
+import { getCookie } from 'lib/cookies';
+
+const CookieNames = {
+    PW_MANAGER_DISMISSED: 'GU_PWMANAGER_DISMISSED',
+    GU_SO: 'GU_SO',
+};
+
+class IdentityFeatures {
+    promptForSignIn: any;
```

Comment:
wouldn't this be a boolean?

Reply:
this was copy pasted from `commercial-features.js`, where the type annotation was what you see here.

Reply:
done: 9980ee9